assets/i18n/hi.json

```diff
@@ -0,0 +1,276 @@
+  "I18N_SIGNUP_LICENSE_OBJECTIVE": "एक <[licenselink]> लाइसेंस के उपयोग से आप एक्सप्लोरेशन्स को  आज़ादी से प्रतिलिपि, पुनः प्रयोग, रीमिक्स तथा  फिर से विभाजित कर सकते है। मुख्य शर्त यह है कि अगर कोई  रीमिक्स तथा फिर से विभाजित कर एक्सप्लोरेशन का परिवर्तन करता है  , तो उन्हें भी उसी लाइसेंस के तहत उनके एक्सप्लोरेशन को वितरित करना होगा।",
+  "I18N_SIGNUP_LOADING": "लोड हो रहा है",
+  "I18N_SIGNUP_PAGE_TITLE": "समुदाय से जुड़े- Oppia",
+  "I18N_SIGNUP_REGISTRATION": "पंजीकरण",
+  "I18N_SIGNUP_SEND_ME_NEWS": "मुझे साइट के बारे में समाचार और अद्यतन भेजें ।",
+  "I18N_SIGNUP_SITE_DESCRIPTION": "<[sitename]>, सीखने के संसाधनों का एक खुला कॉमन्स है. इस पर सभी सामग्री को स्वतंत्र रूप से पुन: प्रयोज्य और साझा किया जा सकता है ।",
+  "I18N_SIGNUP_SITE_OBJECTIVE": "<[sitename]> उच्च गुणवत्ता सीखने के संसाधनों का एक सेट के लगातार सुधार को बढ़ावा देता है जो की स्वतंत्र रूप से सभी के लिए उपलब्ध हैं ।",
+  "I18N_SIGNUP_UPDATE_WARNING": "कृपया ध्यान दें कि हमने हाल ही में हमारी उपयोग की शर्तों को नवीनीकृत किया है ।",
+  "I18N_SIGNUP_USERNAME": "उपयोगकर्ता नाम",
+  "I18N_SIGNUP_USERNAME_EXPLANATION": "आपका उपयोगकर्ता नाम आपके योगदान के बगल में दिखाया जाएगा।",
+  "I18N_SIGNUP_WAIVER_OBJECTIVE": "वेवर ऑफ़ थे अत्त्रिबुतिओं  (BY) रेक्विरेमेंट अर्थात, यदि कोई इस काम को दोबारा इस्तेमाल करता है , उन्हें लेखकों को विशेषता देने की कोई आवस्यकता नहीं हैं। तथापि, आपके व्यक्तिगत योगदान के सभी विवरण साइट के एक्सप्लोरेशन लोग पर उपलब्ध रहेगी । जो लोग एक्सप्लोरेशन का पुनः प्रयोग करते है उन्हें हम प्रोत्साहित (आवश्यक नहीं) करते है की वे अपने काम में इस पेज की एक लिंक जोड़े ।",
+  "I18N_SIGNUP_WHY_LICENSE": "क्यों CC-BY-SA?",
+  "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION": "एक्सप्लोरेशन इंटरैक्टिव सबक है जो स्थिर वीडियो या पाठ की तुलना में अधिक गहरा अनुभव प्रदान करते है और सीखने में मदद करते है ।",
+  "I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION": "Oppia एक मुफ्त अथवा  ओपन सोर्स मंच है जो इंटरैक्टिव गतिविधियों से भरा है जिसे हम एक्सप्लोरेशन्स कहते है। अफसोस की बात है, Oppia को ठीक ढंग से काम करने के लिए आपके ब्राउज़र पे जावास्क्रिप्ट की जरुरत है  परंतु आपके ब्राउज़र पे जावास्क्रिप्ट उपलब्ध नहीं है। अगर आपको जावास्क्रिप्ट हेतु मदद की जरुरत है, <a href=\"<[hrefUrl]>\">यहां क्लिक करे</a>",
+  "I18N_SPLASH_JAVASCRIPT_ERROR_THANKS": "धन्यवाद।",
+  "I18N_SPLASH_JAVASCRIPT_ERROR_TITLE": "हमें आपके ब्राउज़र के जावास्क्रिप्ट की आवश्यकता है।",
+  "I18N_SPLASH_PAGE_TITLE": "Oppia: सिखाओ, जानो और एक्स्प्लोर करो।",
+  "I18N_SPLASH_SECOND_EXPLORATION_DESCRIPTION": "एक्सप्लोरेशन्स बनाना काफी आसान है। वे आसानी से व्यक्तिगत छात्र प्रतिक्रिया और दुनिया भर में अपने शिक्षार्थियों के अनुभव के अनुसार अनुकूलित कर रहे हैं ।",
+  "I18N_SPLASH_SITE_FEEDBACK": "साइट फीडबैक",
+  "I18N_SPLASH_SUBTITLE": "Oppia के द्वारा आसानी से इंटरैक्टिव एक्सप्लोरेशन्स बनाया जा सकता है जो सिखाता है अथवा संलग्न करता है।",
+  "I18N_SPLASH_THIRD_EXPLORATION_DESCRIPTION": "Oppia के द्वारा कई विषयो पर एक्सप्लोरेशन्स बनाया और शेयर किया जा सकता है , जो केवल आपकी कप्लना द्वारा सिमित है।",
```

Comment:
I think the first sentence here should say
"Oppia के द्वारा कई विषयो पर एक्सप्लोरेशन्स बनाये और शेयर किये जा सकते  है" since it is plural.